write-rustdoc-hide-lines/src/utils.rs

```diff
@@ -0,0 +1,25 @@
+use anyhow::Result;
+use std::{
+    ffi::OsStr,
+    fs::{self, DirEntry},
+    path::Path,
+};
+
+pub fn visit_dir_md_files(dir: &Path, cb: &dyn Fn(&DirEntry) -> Result<()>) -> Result<()> {
```

Comment:
I don't think this really need it's own file, could just be declared directly in the formatter file.

Reply:
Changed in 7bbd5cf.